generate/generate_report_test.go

```diff
@@ -250,10 +252,27 @@ func assertReportHasError(t *testing.T, report generate.Report, err error) {
 
 func assertEqualReports(t *testing.T, got, want generate.Report) {
 	t.Helper()
-	if diff := cmp.Diff(got, want); diff != "" {
-		t.Errorf("got %s", got)
-		t.Errorf("want %s", want)
-		t.Errorf("got(-) want(+)")
+
+	// WHY: we can't just use cmp.Diff since the errors included on the Report
```

Comment:
@kassianh this should solve the problem you were having testing generate file :smiley_cat: